spec/lib/mcb/editor/courses_editor_spec.rb

```diff
@@ -13,12 +13,15 @@ def run_editor(*input_cmds)
   let(:email) { 'user@education.gov.uk' }
   let(:provider) { create(:provider, provider_code: provider_code) }
   let(:accredited_body) { create(:provider, :accredited_body) }
-  let!(:mathematics) { find_or_create(:ucas_subject, :mathematics) }
-  let!(:biology) { find_or_create(:ucas_subject, subject_name: "Biology") }
-  let!(:secondary) { find_or_create(:ucas_subject, :secondary) }
+  let!(:japanese) { find_or_create(:subject, subject_name: 'Japanese', type: :ModernLanguagesSubject) }
+  let!(:mathematics) { find_or_create(:subject, :mathematics, type: :PrimarySubject) }
+  let!(:biology) { find_or_create(:subject, subject_name: 'Biology', type: :SecondarySubject) }
+  let!(:english) { find_or_create(:subject, :english, type: :FurtherEducationSubject) }
   let(:current_cycle) { RecruitmentCycle.current_recruitment_cycle }
   let!(:next_cycle) { find_or_create(:recruitment_cycle, year: '2020') }
   let(:is_send) { false }
+  let(:subjects) { [english, biology] }
```

Comment:
can we refrain from having tests that makes a course that don't fit in the rules, unless its to test the rules